viper/modules/pehash/pehasher.py

```diff
@@ -1,37 +1,16 @@
-# This file is part of Viper - https://github.com/botherder/viper
+# -*- coding: utf-8 -*-
+# This file is part of Viper - https://github.com/viper-framework/viper
 # See the file 'LICENSE' for copying permission.
 
 from __future__ import division
 
-import sys
 import bz2
-import string
 import hashlib
+import pefile
+import bitstring
 
-try:
```

Comment:
Okay, no need to change it then, as long as it doesn't raise an exception, I'm cool with it.